code/game/alternate_appearance.dm

```diff
@@ -202,3 +202,17 @@ GLOBAL_LIST_EMPTY(active_alternate_appearances)
 
 /datum/atom_hud/alternate_appearance/basic/heretics/mobShouldSee(mob/M)
 	return IS_HERETIC(M) || IS_HERETIC_MONSTER(M)
+
+/datum/atom_hud/alternate_appearance/basic/one_person
```

Comment:
We have an identical object to this.
`/datum/atom_hud/alternate_appearance/basic/onePerson`

Repath it if you really want

Reply:
repathed